docs/development/api-wrapper/classes/topbar.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Topbar
+description: Create buttons in the top bar.
+---
+
+Create buttons in the top bar, next to the navigation buttons.
+
+This is useful for creating buttons which actions has an impact on the whole app and is generally static, such as a button that opens a settings menu.
```

Comment:
```suggestion
This is useful for creating buttons that are generally static and whose actions have an impact on the whole app, such as a button that opens a settings menu.
```
Am I understanding this statement correctly? I think this would be a clearer way of wording it.